spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -388,9 +389,63 @@ public class MeteredBean {
 }
 ----
 
+==== @SimplyTimed
+An annotation for marking a constructor or method of an annotated object as simply timed.
+The metric of type SimpleTimer tracks how frequently the annotated object is invoked, and tracks how long it took the invocations to complete.
```

Comment:
"how frequently" implies to me a connection to meters, which measure how frequently something is called (in invocations per second). I'd change this to something akin to 'tracks the count of invocations of the annotated object'